frontend/src/data/institutions.ts

```diff
@@ -0,0 +1,7 @@
+const institutions = [
+  'Freie Universität Berlin',
+  'FAU Erlangen-Nürnberg',
+  'PH Schwyz',
+]
+
+export default institutions
```

Comment:
```suggestion
export const institutions = [
  'Freie Universität Berlin',
  'FAU Erlangen-Nürnberg',
  'PH Schwyz',
] as const
```
We should use named exports and make the array immutable

Reply:
Makes absolute sense, done!